package.json

```diff
@@ -55,21 +57,38 @@
 		"signal-exit": "^3.0.2",
 		"slice-ansi": "^3.0.0",
 		"string-length": "^3.1.0",
+		"type-fest": "0.12.0",
+		"typescript": "^3.8.3",
 		"widest-line": "^3.1.0",
 		"wrap-ansi": "^6.2.0",
-		"yoga-layout-prebuilt": "^1.9.3"
+		"yoga-layout-prebuilt": "1.9.5"
```

Comment:
I think it was the result of calling `npm install <dep>`. It was not intentional. The only one that's intentional is `ts-node` because after 7.0.0 it got slower.